src/components/HeaderWithBackButton/headerWithBackButtonPropTypes.js

```diff
@@ -84,6 +84,9 @@ const propTypes = {
 
     /** Children to wrap in Header */
     children: PropTypes.node,
+
+    /** Single execution function to wrap the action in */
```

Comment:
May be this is better
```suggestion
    /** Single execution function to prevent concurrent navigation actions */
```